### PR TITLE
Release 1.3.0

### PR DESCRIPTION
**Pull request by ericbutz**

### CODE REVIEW

File: src/entities/AccountResponse.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
```
Comment (andreyvEze) [inline src/entities/AccountResponse.ts]: could we remove this empty lines

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@fioprotocol/fiosdk",
-    "version": "1.2.1",
+    "version": "1.3.0",
```
Comment (andreyvEze) [inline package.json:3]: want just to confirm why

File: src/entities/LocksResponse.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import {LockPeriod} from './LockPeriod'
```
Comment (andreyvEze) [inline src/entities/LocksResponse.ts]: spaces

File: tests/index.spec.js [hunk 4]
```diff
@@ -118,7 +138,8 @@ describe('Testing generic actions', () => {
 
   const newFioDomain = generateTestingFioDomain()
   const newFioAddress = generateTestingFioAddress(newFioDomain)
-  const pubKeyForTransfer = FIOSDK.derivedPublicKey('5HvaoRV9QrbbxhLh6zZHqTzesFEG5vusVJGbUazFi5xQvKMMt6U')
+  const publicKeyRes = FIOSDK.derivedPublicKey('5HvaoRV9QrbbxhLh6zZHqTzesFEG5vusVJGbUazFi5xQvKMMt6U')
+  pubKeyForTransfer = publicKeyRes.publicKey
```
Comment (andreyvEze) [inline tests/index.spec.js]: should be `const` here

File: tests/index.spec.js [hunk 5]
```diff
@@ -63,7 +64,8 @@ before(async () => {
     baseUrl,
     fetchJson
   )
-  testFioAddressName = generateTestingFioAddress()
+  testDomain = generateTestingFioDomain()
```
Comment (andreyvEze) [inline tests/index.spec.js]: `const`

File: tests/index.spec.js [hunk 6]
```diff
@@ -89,10 +91,28 @@ before(async () => {
   await timeout(receiveTransferTimout)
 
   try {
+
     const isAvailableResult = await fioSdk.genericAction('isAvailable', {
-      fioName: testFioAddressName
+      fioName: testDomain
     })
     if (!isAvailableResult.is_registered) {
+      result = await fioSdk.genericAction('registerFioDomain', {
```
Comment (andreyvEze) [inline tests/index.spec.js]: `const`

File: tests/index.spec.js [hunk 7]
```diff
@@ -89,10 +91,28 @@ before(async () => {
...
+        fioDomain: testDomain,
+        maxFee: defaultFee
+      })
+    }
+
+    result = await fioSdk.genericAction('setFioDomainVisibility', {
```
Comment (andreyvEze) [inline tests/index.spec.js]: `const` / `let`

File: tests/testnet.spec.js [hunk 8]
```diff
@@ -710,30 +764,27 @@ describe('Request funds, cancel funds request', () => {
...
-    expect(pendingReq.fio_request_id).to.be.a('number')
-    expect(pendingReq.fio_request_id).to.equal(requestId)
-    expect(pendingReq.payer_fio_address).to.be.a('string')
-    expect(pendingReq.payer_fio_address).to.equal(testFioAddressName)
-    expect(pendingReq.payee_fio_address).to.be.a('string')
-    expect(pendingReq.payee_fio_address).to.equal(testFioAddressName2)
-  } catch (e) {
-    console.log(e);
-  }
+      await timeout(4000)
```
Comment (andreyvEze) [inline tests/testnet.spec.js:767]: lets format a little

**Review by 0xCasey — APPROVED** [review]

**Pull request merged**